backend/routes_departments.py

```
@@ -0,0 +1,156 @@
+    def get_departments():
+        """Get all departments"""
+        try:
+            include_inactive = request.args.get('include_inactive', 'false').lower() == 'true'
+            
+            if include_inactive:
+                departments = Department.query.all()
+            else:
+                departments = Department.query.filter_by(is_active=True).all()
+            
+            return jsonify([dept.to_dict() for dept in departments])
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)